src/axom/quest/examples/quest_signed_distance_interface.cpp

```diff
@@ -91,22 +147,36 @@ int main ( int argc, char** argv )
   initialize_logger( );
 
   // STEP 2: parse command line arguments
-  parse_args( argc, argv );
+  Arguments args;
+  CLI::App app {"Driver for signed distance query"};
+
+  try
+  {
+    args.parse(argc, argv, app);
+  }
+  catch (const CLI::ParseError &e)
+  {
+    // TOOD convert to MPI parsing!
```

Comment:
It is a change I have to make before merging.  

Since this app could be using MPI, I have to modify it so it cleans up MPI in case of a parsing error.